2-mandatory/1-weather-report.js

```diff
@@ -12,52 +12,55 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
-}
+  // TODO
+
+  const temperatureStatementArray = [];
 
+  for (let i = 0; i < cities.length; i++) {
+    const city = cities[i];
+    const temperature = temperatureService(city);
+    const statement = `The temperature in ${city} is ${temperature} degrees`;
+    temperatureStatementArray.push(statement);
+  }
```

Comment:
excellent way to do that.

Reply:
I learned a lotfrom that Udemy tutorial, he always uses template literal, you have watched it too